features/steps/fhir.py

```diff
@@ -144,7 +145,8 @@ def step_impl(context):
 @given('there is at least 1 entry')
 def step_impl(context):
     resource = context.response.json()
-    entries = resource.get('entry', [])
+    entries = [entry['resource'] for entry in resource.get('entry', [])
```

Comment:
Technically this property is optional, so I am thinking that a more precise logical assessment could be to look for resources of the appropriate type, with either a mode=match or an empty mode.